rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -29,438 +29,6 @@ namespace pir {
 
 namespace {
 
-/*
```

Comment:
did you move this stuff unchanged or are there changes in the code you moved into the other file?